systemd/hyprland-session.service.in

```diff
@@ -9,8 +9,8 @@ After=graphical-session-pre.target
 
 [Service]
 Type=notify
-ExecStart=/usr/bin/Hyprland
-ExecStop=/usr/bin/hyprctl dispatch exit
-ExecStopPost=/usr/bin/systemctl --user unset-environment DISPLAY WAYLAND_DISPLAY HYPRLAND_INSTANCE_SIGNATURE XDG_CURRENT_DESKTOP
+ExecStart=@CMAKE_INSTALL_PREFIX@@prefix@/bin/Hyprland
+ExecStop=@CMAKE_INSTALL_PREFIX@@prefix@/bin/hyprctl dispatch exit
+ExecStopPost=@CMAKE_INSTALL_PREFIX@@prefix@/bin/systemctl --user unset-environment DISPLAY WAYLAND_DISPLAY HYPRLAND_INSTANCE_SIGNATURE XDG_CURRENT_DESKTOP
```

Comment:
This line is not correct, as systemd doesn't live in Hyprland's prefix. It should simply be `systemctl ...` and systemd should find it through PATH.